### PR TITLE
feat: make tmp dir optional

### LINKED ISSUE #4: dir option
`/tmp` isn't accessible in all environements

### PR DESCRIPTION
**Pull request by fengmk2**
Fixes #4

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -50,6 +52,30 @@ describe('Co Multipart', function () {
...
+      assert.equal(parts.files.length, 2)
+      assert.equal(Object.keys(parts.file).length, 2)
+      parts.files.forEach(function (file) {
+        assert.equal(file.path.indexOf(tmpdir), 0)
+      })
+
+      rimraf.sync(tmpdir)
+    })
+    .then(done)
+    .catch(done)
```
Comment (coderhaoxin) [inline test.js]: why not just `return co(...)`

**Comment by coderhaoxin** [comment]
add doc?

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Deletes all the files in this multipart request.
 
 The MIT License (MIT)
 
-Copyright (c) 2013 Jonathan Ong me@jongleberry.com
+Copyright (c) 2013 - 2015 Jonathan Ong me@jongleberry.com
```
Comment (coderhaoxin) [inline README.md]: Is this necessary?

Comment (fengmk2) [inline README.md]: revert

**Comment by fengmk2** [comment]
test fix and doc add.

**Comment by coderhaoxin** [comment]
0.2.0 published!

**Comment by fengmk2** [comment]
:+1: 

I think 1.0.0 will be better.

**Pull request merged**